CHANGELOG/CHANGELOG-1.5.md

```diff
@@ -0,0 +1,98 @@
+- Add support for IPPool and IP annotations on Pod and PodTemplate of Deployment and StatefulSet in AntreaIPAM mode. ([#3093](https://github.com/antrea-io/antrea/pull/3093) [#3042](https://github.com/antrea-io/antrea/pull/3042) [#3141](https://github.com/antrea-io/antrea/pull/3141) [#3164](https://github.com/antrea-io/antrea/pull/3164) [#3146](https://github.com/antrea-io/antrea/pull/3146), [@gran-vmv] [@annakhm])
+  * IPPool annotation on Pod has a higher priority than the IPPool annotation on Namespace
+  * A StatefulSet Pod's IP will be kept after Pod restarts when the IP is allocated from IPPool
+  * Refer to [Antrea IPAM Capabilities] for more information
+- Implement secondary network for Pods using SRIOV interface for CNF use case. ([#2651](https://github.com/antrea-io/antrea/pull/2651), [@arunvelayutham])
+- Add support for allocating external IPs for LoadBalancer Services from ExternalIPPool which was used by Egress. ([#3147](https://github.com/antrea-io/antrea/pull/3147) [@Shengkai2000]) [Alpha - Feature Gate: `ServiceExternalIP`]
+- Add support for antctl in the flow aggregator Pod. ([#2878](https://github.com/antrea-io/antrea/pull/2878), [@yanjunz97])
+  * Support `antctl get log-level` for changing log verbosity level
+  * Support `antctl get flowrecords [-o json]` for dumping flow records
+  * Support `antctl get recordmetrics` for dumping flow records metrics
+- Add support for the "Pass" action in Antrea-native policies to skip evaluation of further Antrea-native policy rules and delegate evaluation to Kubernetes NetworkPolicy. ([#2964](https://github.com/antrea-io/antrea/pull/2964), [@Dyanngg])
+- Add user documentation for using Project Antrea with [Fluentd] in order for efficient audit logging. ([#2853](https://github.com/antrea-io/antrea/pull/2853), [@qiyueyao])
+- Add user documentation for [deploying Antrea on AKS Engine]. ([#2963](https://github.com/antrea-io/antrea/pull/2963), [@jianjuns])
+- Improve [NodePortLocal documentation] to list supported Service types and add information about existing integrations with external Load Balancers. ([#3113](https://github.com/antrea-io/antrea/pull/3113), [@antoninbas])
+- Document how to run Antrea e2e tests on an existing K8s cluster ([#3045](https://github.com/antrea-io/antrea/pull/3045), [@xiaoxiaobaba])
+
+### Changed
+
+- Make LoadBalancer IP proxying configurable for AntreaProxy to support scenarios in which Pod-to-ExternalIP traffic is desired to be sent to the external LoadBalancer. ([#3130](https://github.com/antrea-io/antrea/pull/3130), [@antoninbas])
```

Comment:
in which it is desirable to send Pod-to-ExternalIP traffic to the external LoadBalancer